### ISSUE TITLE
Russian Translation for Playnite

### ISSUE THREAD

**Issue by Evene174** [JosefNemec/Playnite#313]
[russian.zip](https://github.com/JosefNemec/Playnite/files/2297138/russian.zip)

<--- Update 17.08.18 for 4.4.

**Comment by evgdovg1996** [comment]
@JosefNemec Missing words in localization file:
![1](https://user-images.githubusercontent.com/26068083/34119209-258b201e-e443-11e7-86a0-700c707c49e6.png)
![2](https://user-images.githubusercontent.com/26068083/34119210-25b1f824-e443-11e7-899c-c7fcf419bae9.png)
![3](https://user-images.githubusercontent.com/26068083/34119211-25d4c250-e443-11e7-84f0-c94f505be633.png)

**Comment by JosefNemec** [comment]
3.2 update added new string for localization:
https://github.com/JosefNemec/Playnite/commit/ad9500c54713b53cf6e8bab1df4f0ee9fbf0be1a#diff-fc6eaa727d5578c3ac3d59652659d9e2
https://github.com/JosefNemec/Playnite/commit/b122a110cce6735adca31c3a5caadd0ab6451f91#diff-fc6eaa727d5578c3ac3d59652659d9e2

**Comment by evgdovg1996** [comment]
@JosefNemec 
https://imgur.com/a/MhOAf
3.2 still has problems with this strings

**Comment by JosefNemec** [comment]
Damn, I missed it on that screenshot, I'll fix it in next version.

**Comment by evgdovg1996** [comment]
@JosefNemec Will you tell me which I should add in xaml for this? And it will be ready until 3.3 release

**Comment by JosefNemec** [comment]
3.3 update added new string for localization:
https://github.com/JosefNemec/Playnite/commit/ec65168e6fb0f6dc207404c78f7647913441e956#diff-fc6eaa727d5578c3ac3d59652659d9e2

**Comment by evgdovg1996** [comment]
@JosefNemec 
Updated for 3.3. 
[russian.zip](https://github.com/JosefNemec/Playnite/files/1580194/russian.zip)
Now fully translated, please add it to next release.

**Comment by evgdovg1996** [comment]
@JosefNemec Updated Russian localization,please add this in next version.
[russian.zip](https://github.com/JosefNemec/Playnite/files/1583518/russian.zip)

**Comment by JosefNemec** [comment]
I won't be posting updates to original English files in every separate localization issue anymore. Please subscribe to new [localization updates](https://github.com/JosefNemec/Playnite/issues/385) issue to be notified when new stuff needs translation. But please keep posting your updated files in appropriate language issues. Thanks

**Comment by Evene174** [comment]
@JosefNemec Updated for 3.5

**Comment by ziv808** [comment]
Hello! I made a new version of the Russian localization for the application version 4.63.
The translation was checked and translated correctly, the program also behaves stably, without errors. (The first time we translated, after we completed the translation, the program did not want to run).

I would also like to emphasize that in the file “english.xaml” there is no phrase “Export Library” from the section “Extensions”. In addition, I would like to add that I really liked the program, so I will localize this program further!
[russian.zip](https://github.com/JosefNemec/Playnite/files/2668368/russian.zip)
